.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -514,8 +537,8 @@
     </Target>
 
     <Target Name="ResizetizeImages"
-        Inputs="$(MSBuildThisFileFullPath);$(_ResizetizerTaskAssemblyName);$(_ResizetizerInputsFile);@(MauiImage)"
-        Outputs="$(_ResizetizerStampFile)"
+        Inputs="$(MSBuildThisFileFullPath);$(_ResizetizerTaskAssemblyName);$(_ResizetizerInputsFile);$(_MauiSplashInputsFile);@(MauiImage)"
+        Outputs="$(_ResizetizerStampFile);$(_MauiSplashStampFile)"
```

Comment:
This may actually fix a bunch of other "splash screens are not updating" issues.

Reply:
fixed in other PRs